### ISSUE TITLE
Crash if /etc/bubblejail does not exists.

### ISSUE THREAD

**Issue by rusty-snake** [igo95862/bubblejail#63]
### Output of `bubblejail --version`

0.8.0

### Your distro name and version

Fedora 38

#### STR

- Install bubblejail in a way that `/etc/bubblejail` is not exacted.
  - I build [my own rpm](https://github.com/rusty-snake/fedora-extras/tree/main/bubblejail) for it. Presumably there is a step in the rpmbuild tools chain that removes empty directories. Maybe filling it with an example or a short README is a good idea.
- Start bubblejail-config and click on "Create instance"

```py3
Traceback (most recent call last):
  File "/usr/lib64/bubblejail/python_packages/bubblejail/bubblejail_gui_qt.py", line 698, in switch_to_create_instance
    container = CreateInstanceWidget(self)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/bubblejail/python_packages/bubblejail/bubblejail_gui_qt.py", line 574, in __init__
    for profiles_directory in \
  File "/usr/lib64/bubblejail/python_packages/bubblejail/bubblejail_directories.py", line 109, in iter_profile_directories
    profiles_dir.mkdir(exist_ok=True)
  File "/usr/lib64/python3.11/pathlib.py", line 1116, in mkdir
    os.mkdir(self, mode)
FileNotFoundError: [Errno 2] Datei oder Verzeichnis nicht gefunden: '/etc/bubblejail/profiles'
```

#### Expected behaviour

1. Does not crash because of a missing directory.
2. Does not crash when it fails to create a system directory (if running as regular user).

**Comment by igo95862** [comment]
Hello @rusty-snake 

The issue is that if a profiles directory is missing bubblejail will try to recreate it. This was first added when there only was user config directory (under `~/.config`) but later a system wide configuration directory was added. This is normally not an issue because the `/etc` directory exists on most installs but I guess Fedora strips empty directories.

**Comment by igo95862** [comment]
Should be fixed in 75c24c3060f53ab27fbac65c0bb3d9ca6b911bdf. The missing directories should now be skipped. Can you give it a try?

**Comment by rusty-snake** [comment]
Works, Thanks.
